addons/l10n_ro_edi/views/account_move_views.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="l10n_ro_edi_view_move_form" model="ir.ui.view">
+        <field name="name">l10n_ro_edi.view.move.form</field>
+        <field name="model">account.move</field>
+        <field name="priority">30</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='journal_div']" position="after">
+                <label for="l10n_ro_edi_state"
+                       invisible="not l10n_ro_edi_state or state == 'draft' or move_type in ('in_invoice', 'in_refund')"/>
+                <div name="l10n_ro_edi_div"
+                     class="d-flex"
+                     invisible="not l10n_ro_edi_state or state == 'draft' or move_type in ('in_invoice', 'in_refund')">
+                    <field name="l10n_ro_edi_state" class="oe_inline"/>
+                </div>
+            </xpath>
+
+            <!-- CIUS-RO Documents Tab -->
+            <xpath expr="//page[@id='other_tab_entry']" position="after">
+                <page id="l10n_ro_edi_documents"
+                      string="E-Factura"
+                      invisible="not l10n_ro_edi_document_ids">
+                    <field name="l10n_ro_edi_document_ids">
+                        <tree create="false" edit="false"
+                              decoration-primary="state == False"
+                              decoration-danger="state == 'error'"
+                              decoration-warning="state in ('sending', 'nok')"
+                              decoration-success="state in ('sent', 'ok')">
+                            <field name="datetime"/>
+                            <field name="state"/>
+                            <button name="action_l10n_ro_edi_fetch_status"
+                                    type="object"
+                                    string="Fetch status"
+                                    invisible="state != 'sending'"/>
+                            <button name="action_l10n_ro_edi_download_answer"
+                                    type="object"
+                                    string="Download answer"
+                                    invisible="state not in ('ok', 'nok')"/>
+                        </tree>
+                    </field>
```

Comment:
That's a very different way to handle that from what we did in Mexico... I don't want all EDIs to be different.
For example, in Mexico, there is a owl widget to show the error message or to retry the sending of the EDI... What not doing the same here?